Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1229,7 +1229,7 @@ extension Tensor {
     /// Returns `true` if all given axes are in the range `[-rank, rank)`.
     @usableFromInline
     internal func areAxesInRange<T: BinaryInteger>(_ axes: [T]) -> Bool {
-        return !axes.contains(where: { !isAxisInRange($0) })
+        return !axes.contains(where: { !isAxisInRange($0) }) && ensureNonDuplicated(axes)
```

Comment:
This line's change doesn't match what the commit message says you're changing, and appears to give the method new semantics that clash with its name.    If this PR is about improving error messages, the changes should just do that.